__tests__/is/fqdn.spec.ts

```diff
@@ -0,0 +1,13 @@
+import { isFqdn } from "../../src/Is";
+
+describe("postive cases", () => {
+  test("it should success", () => {
+    expect(isFqdn("foo.com")).toBeTruthy();
```

Comment:
AFAIK, the fqdn includes the subdomain/hostname for instance `www.foo.com`

Reply:
it should be valid, because foo in there also considered as subdomain of com